app/assets/javascripts/preferences/panes/listed/BlogItem.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button } from '@/components/Button';
+import { HorizontalSeparator } from '@/components/shared/HorizontalSeparator';
+import { LinkButton, Subtitle } from '@/preferences/components';
+import { WebApplication } from '@/ui_models/application';
+import { Action, SNComponent, SNItem } from '@standardnotes/snjs/dist/@types';
+import { JSXInternal } from 'preact/src/jsx';
+import React, { useState } from 'react';
```

Comment:
Since we're using Preact, it's fine to delete this line and just import `useState` from preact.
Additionally, `<React.Fragment> ... </React.Fragment>` can be replaced by `<> ... </>`